dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java

```diff
@@ -79,6 +81,13 @@ public void synchronousWrite(Object event) {
                 (loggerName.equalsIgnoreCase(LoggingConstants.SYSTEM_OUT) || loggerName.equalsIgnoreCase(LoggingConstants.SYSTEM_ERR))) {
                 sysLogHolder.getOriginalStream().println(messageOutput);
             }
+
+            //Other wise we will be writing out  accessLog or ffdc or trace
```

Comment:
seems like this comment is backwards... this addition is to allow accessLog / ffdc / trace to appear in console output, correct?